pkg/tfshim/schema/schema.go

```diff
@@ -168,3 +168,85 @@ func (m SchemaMap) Set(key string, value shim.Schema) {
 func (m SchemaMap) Delete(key string) {
 	delete(m, key)
 }
+
+func FromSchema(s shim.Schema) *Schema {
```

Comment:
Where are the additions to this file used?

Reply:
Oops I meant to remove it. I didn't end up using this. Was thinking it can help writing test.